services/recordandreplay.md

```diff
@@ -0,0 +1,26 @@
+# Record and Replay
+
+    identifier: 0x1b72bf50
+    group: sound
+    tags: 8bit, padauk
+    
+A record and replay module. You can record a few seconds of audio and play it back.
```

Comment:
You probably want to document status code values so that you can use the status changed event to notify when the device is I. Recording/replay mode.